spec/model/kubernetes/kubernetes_cluster_spec.rb

```diff
@@ -41,4 +41,31 @@
       expect(kc.valid?).to be true
     end
   end
+
+  describe "#kubeconfig" do
+    kubeconfig = <<~YAML
+      apiVersion: v1
+      kind: Config
+      users:
+        - name: admin
+          user:
+            client-certificate-data: "mocked_cert_data"
+            client-key-data: "mocked_key_data"
+    YAML
+    let(:sshable) { instance_double(Sshable) }
+    let(:vm) { instance_double(Vm, sshable: sshable) }
+    let(:cp_vms) { [vm] }
+
+    it "removes client certificate and key data from users and adds an RBAC token to users" do
+      expect(kc).to receive(:cp_vms).and_return(cp_vms).twice
```

Comment:
while you can do .twice in some situations, usually `.at_least(:once)` is just a lot more robust and loses nothing. Too bad it's wordy. I wouldn't go so far to say that rspec should have made `at_least(:once)` the default behavior, but it's close. Surely it should have a shorter-hand.